lib/rubocop/cop/lint/ordered_magic_comments.rb

```diff
@@ -0,0 +1,86 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Lint
+      #
+      # This cop checks coding magic comment should be first line,
+      # or next of shebang.
+      #
+      # @example
+      #   # bad
+      #
+      #   # frozen_string_literal: true
+      #   # coding: ascii
```

Comment:
This reminded me that I also using `encoding`, not `coding`. Probably we should add some check for this, although it's not a big deal in the age of `utf-8` by default.